app/src/lib/stores/helpers/branch-pruner.ts

```diff
@@ -28,6 +29,33 @@ const ReservedRefs = [
   'refs/heads/release',
 ]
 
+const BranchRefPrefix = `refs/heads/`
+
+/**
+ * Behavior flags for the branch prune execution, to aid with testing and
+ * verifying locally.
+ */
+type PruneRuntimeOptions = {
```

Comment:
I like that this type makes the logic for testing locally a bit easier to digest.